nginx.conf

```diff
@@ -4,7 +4,7 @@ events {
 
 http {
 	server {
-		listen 6601 ssl;
+		listen 443 ssl;
```

Comment:
Just curious why the port is changed?

Reply:
443 is more typical for secured web connections, 6601 was used during testing but is no longer used.

Reply:
Okay, thanks